packages/backend-app-api/src/services/implementations/httpRouter/httpRouterServiceFactory.ts

```diff
@@ -63,6 +73,14 @@ export const httpRouterServiceFactory = createServiceFactory(
         },
         addAuthPolicy(policy: HttpRouterServiceAuthPolicy): void {
           credentialsBarrier.addAuthPolicy(policy);
+          if (
+            policy.allow === 'user-cookie' &&
+            !hasRegistedCookieAuthRefreshMiddleware
+          ) {
+            // Only add the cookie refresh middleware once
+            hasRegistedCookieAuthRefreshMiddleware = true;
+            router.use(createCookieAuthRefreshMiddleware({ auth, httpAuth }));
```

Comment:
We'll need to this in a way where ordering of these calls don't matter, in particular making sure this always takes precedence over any plugin routes. I do wonder if we should just always have it installed though? It's a bit simpler

Reply:
Modified to always have it.